docs/services/gtfs-ckan-uploader.md

```diff
@@ -1,3 +0,0 @@
-# GTFS CKAN Uploader
```

Comment:
why delete this?

Reply:
we deleted the code awhile ago but missed the docs